samples/contrib/ibm-samples/openscale/README.md

```diff
@@ -67,8 +64,8 @@ def secret_pipeline(
                   url=CONFIG_FILE_URL,
                   name=secret_name
   )
-compiler.Compiler().compile(secret_pipeline,  'secret_pipeline.tar.gz')
-run = client.run_pipeline(exp.id, 'secret_pipeline', 'secret_pipeline.tar.gz')
+
+kfp.Client().create_run_from_pipeline_func(secret_pipeline, arguments={})
```

Comment:
Similar as above. Shall we specify an experiment name?

Reply:
I think it's fine to omit it as it's no longer required. https://github.com/kubeflow/pipelines/pull/2293/files#r331666697